services/provider/pb/storageclient.go

```diff
@@ -42,3 +42,13 @@ func (o *OnboardConsumerRequest) SetClientOperatorVersion(version string) ifaces
 	o.ClientOperatorVersion = version
 	return o
 }
+
+func (o *OnboardConsumerRequest) SetClusterID(clusterID string) ifaces.StorageClientOnboarding {
+	o.ClusterId = clusterID
+	return o
+}
+
+func (o *OnboardConsumerRequest) SetNamespacedName(namespacedName string) ifaces.StorageClientOnboarding {
+	o.NamespacedName = namespacedName
+	return o
+}
```

Comment:
Aren't we missing implementations? I believe that without them  `OnboardConsumerRequest` would not satisfy the 
`ifaces.StorageClientOnboarding` interface.

Reply:
All the getters are in [provider.pb.go](https://github.com/raaizik/ocs-operator/blob/rdr-odfinfo-server/services/provider/pb/provider.pb.go#L122-L155)

Reply:
No, that's where the existing ones were and I just added mine. Can move all of them to this file.

Reply:
@leelavg that's incorrect. The `make build` command executes `gen-protobuf` and nothing of this sort got auto-generated, hence why I needed to manually implement it. There's no sense, nor is it recommended, in editing the pb file, that's why I suggested moving them to this file.